instrumentation/spring/spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -279,6 +279,33 @@
       "description": "Enables the DB statement sanitization.",
       "defaultValue": true
     },
+    {
+      "name": "otel.instrumentation.common.peer-service-mapping",
+      "type": "java.util.Map<java.lang.String, java.lang.String>",
+      "description": "Used to specify a mapping from host names or IP addresses to peer services, as a comma-separated list of <code>host_or_ip=user_assigned_name</code> pairs. The peer service is added as an attribute to a span whose host or IP address match the mapping. See https://opentelemetry.io/docs/zero-code/java/agent/configuration/#peer-service-name."
+    },
+    {
+      "name": "otel.instrumentation.http.client.capture-request-headers",
+      "type": "java.util.List<java.lang.String>",
+      "description": "List of HTTP request headers to capture in HTTP clients."
+    },
+    {
+      "name": "otel.instrumentation.http.client.capture-response-headers",
+      "type": "java.util.List<java.lang.String>",
+      "description": "List of HTTP response headers to capture in HTTP clients."
+    },
+    {
+      "name": "otel.instrumentation.http.client.emit-experimental-telemetry",
+      "type": "java.lang.Boolean",
+      "description": "Enable the capture of experimental HTTP client telemetry. Add the <code>http.request.body.size</code> and <code>http.response.body.size> attributes to spans, and record the <code>http.client.request.size</code> and <code>http.client.response.size</code> metrics.",
+      "defaultValue": false
+    },
+    {
+      "name": "otel.instrumentation.http.known-methods",
+      "type": "java.util.List<java.lang.String>",
+      "description": "Configures the instrumentation to recognize an alternative set of HTTP request methods. All other methods will be treated as <code>_OTHER</code>.",
+      "defaultValue": "CONNECT,DELETE,GET,HEAD,OPTIONS,PATCH,POST,PUT,TRACE"
+    },
```

Comment:
`otel.instrumentation.http.server.capture-request-headers` and `otel.instrumentation.http.server.capture-response-headers` are missing

Reply:
server part is added in https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/11667 (instrumentation and metadata)